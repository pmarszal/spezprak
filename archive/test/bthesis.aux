\relax 
\providecommand\hyper@newdestlabel[2]{}
\catcode `"\active 
\bibstyle{plainnat}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\select@language{ngerman}
\@writefile{toc}{\select@language{ngerman}}
\@writefile{lof}{\select@language{ngerman}}
\@writefile{lot}{\select@language{ngerman}}
\select@language{ngerman}
\@writefile{toc}{\select@language{ngerman}}
\@writefile{lof}{\select@language{ngerman}}
\@writefile{lot}{\select@language{ngerman}}
\select@language{ngerman}
\@writefile{toc}{\select@language{ngerman}}
\@writefile{lof}{\select@language{ngerman}}
\@writefile{lot}{\select@language{ngerman}}
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\select@language{ngerman}
\@writefile{toc}{\select@language{ngerman}}
\@writefile{lof}{\select@language{ngerman}}
\@writefile{lot}{\select@language{ngerman}}
\gdef \LT@i {\LT@entry 
    {1}{97.35696pt}\LT@entry 
    {1}{274.75267pt}\LT@entry 
    {1}{54.68173pt}}
\gdef \LT@ii {\LT@entry 
    {1}{97.35696pt}\LT@entry 
    {1}{274.75267pt}\LT@entry 
    {1}{54.68173pt}}
\gdef \LT@iii {\LT@entry 
    {1}{97.35696pt}\LT@entry 
    {1}{329.43442pt}}
\gdef \LT@iv {\LT@entry 
    {1}{97.35696pt}\LT@entry 
    {1}{329.43442pt}}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Einleitung}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\citation{Menz97}
\citation{Horn90,DINEN6232,Menz97,Knuth84}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}Grundlagen}{3}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{eq:pythagoras}{{\relax 2.1}{3}{Grundlagen}{equation.2.0.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2.1}Unterkapitel Gliederungsebene 2}{3}{section.2.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1.1}Unterkapitel Gliederungsebene 3}{3}{subsection.2.1.1}}
\@writefile{lot}{\contentsline {table}{\numberline {\relax 2.1}{\ignorespaces Tabellenbeschreibung}}{3}{table.2.1}}
\newlabel{tab:tabellenplatzhalter}{{\relax 2.1}{3}{Tabellenbeschreibung}{table.2.1}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {\relax 2.1}{\ignorespaces Bildbeschreibung}}{4}{figure.2.1}}
\newlabel{fig:bildplatzhalter}{{\relax 2.1}{4}{Bildbeschreibung}{figure.2.1}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {3}Experimentelle Vorgehensweise}{5}{chapter.3}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {4}Ergebnisse}{7}{chapter.4}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {5}Diskussion}{9}{chapter.5}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {5.1}Unterkapitel}{9}{section.5.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {5.1.1}Unterkapitel}{9}{subsection.5.1.1}}
\@writefile{toc}{\contentsline {chapter}{\numberline {6}Zusammenfassung}{11}{chapter.6}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {A}erster Anhang}{13}{appendix.A}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {B}zweiter Anhang}{15}{appendix.B}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\bibdata{bthesis_datenbank}
\global\@altsecnumformattrue
