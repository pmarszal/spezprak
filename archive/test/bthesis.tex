\NeedsTeXFormat{LaTeX2e}[2005/12/01]
%%    2009/03/12 v1.0 GAUBM Vorlage für Aschlussarbeiten Physik
%% Template fuer Bachelor- und Masterarbeiten
%% an der Fakultaet fuer Physik (c) Thomas Pruschke der GA Universität
%% Verbesserungsvorschlaege bitte an pruschke@theorie.physik.uni-goettingen.de
%%
%% Benoetigte Pakete: datenumber
%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%% Bitte vor dem Veraendern diese Datei umbenennen! %%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% scrbook - Ersatz für LaTeX book Klasse aus dem KOMA Script
%% Moegliche Optionen: diejenigen der Klasse scrbook ausser titlepage

%% deutsche Arbeit:
\documentclass[bachelor,       %% Typ der Arbeit: bachelor oder master
               twoside,        %% zweiseitiges Layout
               BCOR10mm,       %% Bindekorrektur 10 mm
%               liststotoc,nomtotoc,bibtotoc, %% Aufnahme der div. Verzeichnisse
                                              %% ins Inhaltsverzeichnis
               english,ngerman, %% Alternativspr. Englisch, Dokumentspr. Deutsch
%               ngerman,english  %% Alternativspr. Deutsch, Dokumentspr. Englisch
%               final,          %% Endversion; draft fuer schnelles Kompilieren
               ]{GAUBM}

\usepackage{setspace}  %% Zur Setzung des Zeilenabstandes
\usepackage{babel}     %% Sprachen-Unterstuetzung
\usepackage{calc}      %% ermoeglicht Rechnen mit Laengen und Zaehlern
\usepackage[T1]{fontenc}       %% Unterstutzung von Umlauten etc.
\usepackage[latin1]{inputenc}  %% 
%% in aktuellem Linux & MacOS X wird standardmaessig UTF8 kodiert!
%\usepackage[utf8]{inputenc}    %% Wenn latin1 nicht geht ...

\usepackage{amsmath,amssymb} %% zusaetzliche Mathe-Symbole

\usepackage{lmodern} %% type1-taugliche CM-Schrift als Variante zur
                     %% "normalen" EC-Schrift
%% Paket fuer bibtex-Datenbanken
\usepackage[comma,numbers,sort&compress]{natbib}
\bibliographystyle{plainnat}

\newcommand{\tabheadfont}[1]{\textbf{#1}} %% Tabellenkopf in Fett
\usepackage{booktabs}                      %% Befehle fuer besseres Tabellenlayout
\usepackage{longtable}                     %% umbrechbare Tabellen
\usepackage{array}                         %% zusaetzliche Spaltenoptionen

%% umfangreiche Pakete fuer Symbole wie \micro, \ohm, \degree, \celsius etc.
\usepackage{textcomp,gensymb}

%\usepackage{SIunits} %% Korrektes Setzen von Einheiten
\usepackage{units}   %% Variante fuer Einheiten

%% Hyperlinks im Dokument; muss als eines der letzten Pakete geladen werden
\usepackage[pdfstartview=FitH,      % Oeffnen mit fit width
            breaklinks=true,        % Umbrueche in Links, nur bei pdflatex default
            bookmarksopen=true,     % aufgeklappte Bookmarks
            bookmarksnumbered=true  % Kapitelnummerierung in bookmarks
            ]{hyperref}

%% Weiter benoetigte Pakete: datenumber
%% Falls dieses Paket nicht in der Installation vorhanden ist,
%% kann es von der Seite mit diesem Template heruntergeladen werden
%% und in einem LaTeX bekanntem Verzeichnis installiert werden (notfalls
%% dem Verzeichnis mit der Arbeit).
\begin{document}
%%
%%                   Ab hier muessen die Anpassungen geschehen
%%
%% Hier den eigenen Namen einsetzen
\ThesisAuthor{Liliane}{Musterfrau}
%% Hier den Geburtsort einsetzen
\PlaceOfBirth{Dortdorf}
%% Titel Arbeit. Das erste Argument ist der deutsche, das zweite der
%% englische Titel.
\ThesisTitle{Hier steht das Thema der Arbeit in deutsch}{Here comes the title of the thesis in english}
%% Erst- und Zweitgutacher/in
%% Ist der/die Betreuer/in nicht identisch mit dem/r Erstgutachter/in,
%% muss diese/r als optionales Argument angegeben werden.
\FirstReferee[Dr.\ \ldots]{Prof.\ Dr.\ \dots}
\Institute{Institut f\"ur Theoretische Physik}
\SecondReferee{Prof.\ Dr.\ \dots}
%% Beginn und Ende des Anfertigungszeitraumes
\ThesisBegin{1}{4}{2009}
\ThesisEnd{15}{7}{2009}
%% DO NOT TOUCH THESE LINES!!!!
\frontmatter
\maketitle
\cleardoublepage
%% Zusammenfassung. Falls nicht gewuenscht, bitte auskommentieren.
\begin{abstract}
  Hier werden auf einer halben Seite die Kernaussagen der Arbeit
  zusammengefasst.
%% Optional: Stichwoerter. Wenn nicht gewuenscht, koennen die beiden
%% folgenden Zeilen geloescht werden
  \bigskip\par
  \textbf{Stichwörter:} Physik, Bachelorarbeit
\end{abstract}
%% So laesst sich in die andere Sprache umschalten (Englisch bzw. Deutsch)
\begin{otherlanguage}{english}
\begin{abstract}
  Here the key results of the thesis can be presented in about
  half a page.
  \bigskip\par
  \textbf{Keywords:} Physics, Bachelor thesis
\end{abstract}
\end{otherlanguage}

%% Ende des Vorspanns
\cleardoublepage
%% Ab hier 1 1/2 facher Zeilenabstand (durch setspace-Paket)
\onehalfspacing
%% Erzeugt Inhaltsverzeichnis
\tableofcontents

%% Hier kann man seine Bezeichnungsweisen erklaeren. Falls nicht
%% benoetigt, bis einschliesslich \end{nomenclature} auskommentieren
\begin{nomenclature}
%% Fuer die Berechnung der Spaltenbreiten muss \usepackage{calc}
%% geladen sein!
\section*{Lateinische Buchstaben}
\noindent
\begin{longtable}[l]{p{0.2\textwidth}p{0.7\textwidth-6\tabcolsep}p{0.1\textwidth}}
  \tabheadfont{Variable}&\tabheadfont{Bedeutung}&\tabheadfont{Einheit}\\\midrule\endhead
  $A$ & Querschnittsfl"ache & $\unit{m^2}$\\
  $c$ & Geschwindigkeit & $\unitfrac{m}{s}$
\end{longtable}
\section*{Griechische Buchstaben}
\begin{longtable}[l]{p{0.2\textwidth}p{0.7\textwidth-6\tabcolsep}p{0.1\textwidth}}
  \tabheadfont{Variable}&\tabheadfont{Bedeutung}&\tabheadfont{Einheit}\\\midrule\endhead
  $\alpha$  & Winkel & $\unit{\degree}$; --\\
  $\varrho$ & Dichte & $\unitfrac{kg}{m^3}$
\end{longtable}
\section*{Indizes}
\begin{longtable}[l]{p{0.2\textwidth}p{0.8\textwidth-4\tabcolsep}}
  \tabheadfont{Index}&\tabheadfont{Bedeutung}\\\midrule\endhead
  m & Meridian\\
  $r$ & Radial
\end{longtable}
\section*{Abk"urzungen}
\begin{longtable}[l]{p{0.2\textwidth}p{0.8\textwidth-4\tabcolsep}}
  \tabheadfont{Abk"urzung}&\tabheadfont{Bedeutung}\\\midrule\endhead
  2D & zweidimensional\\
  3D & dreidimensional\\
  max & maximal
\end{longtable}
\end{nomenclature}
%% \listoftables und \listoffigures sollten nur bei genuegender Anzahl Tabellen
%% verwendet werden
%\listoffigures
%\listoftables

\mainmatter   %% Anfang Hauptteil

\chapter{Einleitung}
Diese Vorlage \verb!GAUBM!
für Bachelor- bzw.\ Masterarbeiten ist eine Überarbeitung der
Vorlage von Simon Dreher für Abschlußarbeiten am
Institut für Mikrosystemtechnologie (IMTEK)
an der Universität Freiburg. Die eigentliche Datei mit der Klassendefinition
ist \verb!GAUBN.cls!, die Sie zusammen mit dieser Datei erhalten haben. Weitere
Dateien sind \verb!datenumber.sty! und die zugehörigen Sprachdefinitionen
\verb!\datenumber*.ldf!. Im Verzeichnis \verb!figures! finden sich die
von der Klasse benötigten Logos (Universität und Physik) sowie Beispielbilder
für die Übersetzung dieser Beispieldatei (\verb!bthesis.tex!).
Sie können diese Datei als Vorlage für Ihre Arbeit nutzen und entsprechend
modifizieren. Bitte denken Sie daran, sie vorher unter einem eigenen Namen
abzuspeichern.
Um die Datei anzupassen, gehen Sie wie folgt vor:

Bei den Parametern zu \verb!\documentclass[...]{GAUBM}! in der Präambel
kann man durch Umschalten
zwischen \verb!english,ngerman! und \verb!ngerman,english! eine
deutsche Arbeit (erste Variante) mit Englisch als Alternativsprache bzw.\ eine
englische Arbeit (zweite Variante) mit deutsch als Alternativsprache
wählen. Im laufenden Text kann man mit 
\begin{verbatim}
\begin{otherlanguage}{english/ngerman}
...
\end{otherlanguage}
\end{verbatim}
zur alternativen Sprache wechseln.

Nach \verb!\begin{document}! müssen zuerst ein paar Befehle mit
Information über die Arbeit aufgerufen werden:
\begin{enumerate}
\item \verb!\ThesisAuthor{Vorname}{Nachname}!: Die Argumente sind der
Vorname und Nachname der Autorin bzw.\ des Autors der Arbeit.
\item \verb!\PlaceOfBirth{Wohnort}!: Der Geburtsort der Autorin bzw.\ des Autors.
\item \verb!\ThesisTitle{Deutscher Titel}{English title}!: Der deutsche und englische Titel der Arbeit gemäß Antrag.
\item \verb!\Institute{Institut}!: Das Institut, an dem die Arbeit angefertigt wurde.
\item \verb!\FirstReferee[Betreuer/in]{Erste/r Gutachter/in}!: Voller Titel
und Name des/r Erstgutachter/in. Ist der Betreuer der Arbeit \emph{nicht}
identisch mit dem/r Erstgutachter/in, so muß der volle Titel und der Name
des/r Betreuer/in als optionales Argument in eckigen Klammern erscheinen.
\item \verb!\SecondReferee{Zweite/r Gutachter/in}!: Voller Titel
und Name des/r Zweitgutachter/in.
\item \verb!\ThesisBegin{Tag}{Monat}{Jahr}!: Datum des Beginns der Anfertigung
der Arbeit gemäß Antrag.
\item \verb!\ThesisEnd{Tag}{Monat}{Jahr}!: Datum der Fertigstellung der Arbeit.
\item Optional kann mit
\begin{verbatim} 
\begin{abstract}
...
\end{abstract}
\end{verbatim}
eine maximal eine halbe Seite lange Zusammenfassung eingefügt werden.

Falls man die Zusammenfassung in der alternativen Sprache verfassen möchte,
dann geht das mit der Befehlsfolge
\begin{verbatim} 
\begin{otherlanguage}{english/ngerman}
\begin{abstract}
...
\end{abstract}
\end{otherlanguage}
\end{verbatim}
\end{enumerate}

\chapter{Grundlagen}
In diesem Kapitel werden die theoretischen Grundlagen erläutert.

Wichtige Gleichungen, die in der Arbeit häufiger zitiert werden,
sollten eine Gleichungsnummer erhalten.
\begin{equation}
  \label{eq:pythagoras}
  a^2+b^2=c^2
\end{equation}
Zum Beispiel wird in Gleichung~\ref{eq:pythagoras} der Satz des Pythagoras
angegeben.

Gerade im Bereich der Grundlagen wird viel Literatur zitiert, z.B.\
\cite{Menz97}. Falls
mehrere Literaturzitate auf einmal zitiert werden, ist folgendes
z.B.\ möglich \cite{Horn90,DINEN6232,Menz97,Knuth84}.

\section{Unterkapitel Gliederungsebene 2}
Hier sollte etwas Text stehen.
\subsection{Unterkapitel Gliederungsebene 3}
Noch ein paar Beispiele zu Abbildungen und Tabellen:

Abbildung~\ref{fig:bildplatzhalter} verdeutlicht \dots

Wie die Abb.~\ref{fig:bildplatzhalter} und
Tab.~\ref{tab:tabellenplatzhalter} verdeutlichen \dots

\begin{figure}
  \centering
  \includegraphics[width=0.5\linewidth]{figures/bild}
  \caption{Bildbeschreibung}
  \label{fig:bildplatzhalter}
\end{figure}

Text\dots
\begin{table}
  \centering
  \begin{tabular}{llll}
    \toprule
    $A$-Wert&$B$-Wert&$C$-Wert&$D$-Wert\\
    \midrule
    aaaaaa&bbbbbbb&cccccc&ddddddd\\
    aaaaaa&bbbbbbb&cccccc&ddddddd\\
    \bottomrule
  \end{tabular}
  \caption{Tabellenbeschreibung}
  \label{tab:tabellenplatzhalter}
\end{table}

Text\dots

\chapter{Experimentelle Vorgehensweise}
Text\dots
\chapter{Ergebnisse}
Text\dots
\chapter{Diskussion}
Text\dots
\section{Unterkapitel}
Text\dots
\subsection{Unterkapitel}
Text\dots
\chapter{Zusammenfassung}
Text\dots

\appendix
\chapter{erster Anhang}
Text\dots
\chapter{zweiter Anhang}
Text\dots

\cleardoublepage
%% Bibliographie. Das Argument muss der Name der BIBTeX-Datenbank stehen.
%% Ein Beispiel fuer eine solche Datenbank finden Sie in bthesis_datenbank.bib
\bibliography{bthesis_datenbank} 

\chapter*{Danksagung}
Dank\dots

%% Dieser Befehl MUSS am Ende stehen und erzeugt die Erklaerung ueber die
%% benutzten Mittel
\Declaration
\end{document}
