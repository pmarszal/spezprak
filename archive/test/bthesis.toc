\select@language {ngerman}
\select@language {ngerman}
\select@language {ngerman}
\select@language {english}
\select@language {ngerman}
\contentsline {chapter}{\numberline {1}Einleitung}{1}{chapter.1}
\contentsline {chapter}{\numberline {2}Grundlagen}{3}{chapter.2}
\contentsline {section}{\numberline {2.1}Unterkapitel Gliederungsebene 2}{3}{section.2.1}
\contentsline {subsection}{\numberline {2.1.1}Unterkapitel Gliederungsebene 3}{3}{subsection.2.1.1}
\contentsline {chapter}{\numberline {3}Experimentelle Vorgehensweise}{5}{chapter.3}
\contentsline {chapter}{\numberline {4}Ergebnisse}{7}{chapter.4}
\contentsline {chapter}{\numberline {5}Diskussion}{9}{chapter.5}
\contentsline {section}{\numberline {5.1}Unterkapitel}{9}{section.5.1}
\contentsline {subsection}{\numberline {5.1.1}Unterkapitel}{9}{subsection.5.1.1}
\contentsline {chapter}{\numberline {6}Zusammenfassung}{11}{chapter.6}
\contentsline {chapter}{\numberline {A}erster Anhang}{13}{appendix.A}
\contentsline {chapter}{\numberline {B}zweiter Anhang}{15}{appendix.B}
