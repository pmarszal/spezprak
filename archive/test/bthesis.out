\BOOKMARK [0][]{nom.0}{Nomenklatur}{}% 1
\BOOKMARK [0][]{chapter.1}{1 Einleitung}{}% 2
\BOOKMARK [0][]{chapter.2}{2 Grundlagen}{}% 3
\BOOKMARK [1][]{section.2.1}{2.1 Unterkapitel Gliederungsebene 2}{chapter.2}% 4
\BOOKMARK [2][]{subsection.2.1.1}{2.1.1 Unterkapitel Gliederungsebene 3}{section.2.1}% 5
\BOOKMARK [0][]{chapter.3}{3 Experimentelle Vorgehensweise}{}% 6
\BOOKMARK [0][]{chapter.4}{4 Ergebnisse}{}% 7
\BOOKMARK [0][]{chapter.5}{5 Diskussion}{}% 8
\BOOKMARK [1][]{section.5.1}{5.1 Unterkapitel}{chapter.5}% 9
\BOOKMARK [2][]{subsection.5.1.1}{5.1.1 Unterkapitel}{section.5.1}% 10
\BOOKMARK [0][]{chapter.6}{6 Zusammenfassung}{}% 11
\BOOKMARK [0][]{appendix.A}{A erster Anhang}{}% 12
\BOOKMARK [0][]{appendix.B}{B zweiter Anhang}{}% 13
